src/ThirdPartyService.php

```diff
@@ -2,13 +2,14 @@
 
 namespace AllDressed\Constants;
 
+use AllDressed\Constants\Concerns\CanBeRandomized;
 use AllDressed\Constants\Concerns\HasTranslation;
 
 enum ThirdPartyService: string
 {
-    use HasTranslation;
+    use HasTranslation, CanBeRandomized;
```

Comment:
Tu peux le mettre en ordre alphabétique. Normalement le CS Fixer devrait le sort automatiquement. 😉

Reply:
Hmm, je l'avais réparé hier soir pour la config de Swoo 🤔 On dirait qu'il n'a propagate sur tout mes projets, I'll give another look so it affects all projects 👍